rfcs/0030-ensure-index-less-than-length-in-since/0030-ensure-index-less-than-length-in-since.md

```diff
@@ -1 +1,87 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/223>
+---
+Number: "0030"
+Category: Consensus (Soft Fork)
+Status: Draft
+Author: Ian Yang
+Organization: Nervos Foundation
+Created: 2021-02-04
+---
+
+# Ensure That Index Is Less Than Length In the Input Since Field Using Epoch With Fraction
+
+## Abstract
+
+This document proposes to fix a bug in the CKB reference implementation. When a transaction input uses the epoch with fraction in the `since` field, the `value` must ensure that the index is less than the length.
```

Comment:
I wouldn't personally say this is a bug in the `CKB reference implementation`. In Lina, a different implementation will also need to replicate this bug for the consensus rules to fully work. In this sense, I would argue this is a bug in the consensus rules, which is derived from the CKB reference implementation.